src/components/SecondaryPanes/XHRBreakpoints.js

```diff
@@ -193,13 +194,13 @@ class XHRBreakpoints extends Component<Props, State> {
         key={path}
         title={path}
         onDoubleClick={(items, options) => this.editExpression(index)}
+        onClick={e => this.handleCheckbox(index, e)}
```

Comment:
Could we maybe achieve this by having a `<label>` wrap the `<input>` and have a `for` attribute?

Reply:
Added changes. No `for` attribute needed since association is implicit.